pid_gains:
  panda_joint1:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_joint2:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_joint3:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_joint4:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_joint5:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_joint6:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_joint7:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_finger_joint1:
    p: 100.0
    i: 0.01
    d: 10.0
  panda_finger_joint2:
    p: 100.0
    i: 0.01
    d: 10.0

